### ISSUE TITLE
Emperor isn't perfectly aligned in empire plots

### ISSUE THREAD

**Issue by fedarko** [biocore/empress#356]
Not a big deal, but now the "31 / 31 visible" text is partially overlapped by the Empress visualization:

![image](https://user-images.githubusercontent.com/4177727/91383142-43898d00-e7e0-11ea-9372-fc70f3e9be97.png)

This is present in the master branch of Empress. Not sure where this came from -- probably some minor CSS that's messing with this.

**Comment by fedarko** [comment]
Realized that the [Empire plots created by McHelper](https://view.qiime2.org/visualization/?type=html&src=https%3A%2F%2Fmchelper.ucsd.edu%2Fdownloads%2Fempress%2Fmaster%2Fempire-biplot.qzv) don't have this problem -- turns out my system's Emperor version was just old; updating it solved this problem. Sorry for the false alarm!

**Comment by fedarko** [comment]
It looks like this is happening again, this time with the McHelper versions of plots? Not sure if something changed recently in Empress or Emperor that messed things up.

![funk](https://user-images.githubusercontent.com/4177727/95528633-874de580-098d-11eb-9389-e2452b9b3dfb.png)

**Comment by ElDeveloper** [comment]
FWIW I am not seeing this issue in Safari or Chrome:

# Safari

<img width="2083" alt="Screen Shot 2020-10-09 at 8 41 31 AM" src="https://user-images.githubusercontent.com/375307/95603626-62985300-0a0b-11eb-87ed-0bd9e22fbe8c.png">


# Chrome

<img width="1792" alt="Screen Shot 2020-10-09 at 8 42 04 AM" src="https://user-images.githubusercontent.com/375307/95603619-5f9d6280-0a0b-11eb-98f9-7351390b1f1c.png">

**Comment by kwcantrell** [comment]
I dont seem to be getting this issue on firefox as well 
![Screenshot from 2020-10-09 08-48-10](https://user-images.githubusercontent.com/28127694/95604312-4cd75d80-0a0c-11eb-9f23-550ec210d27f.png)

**Comment by fedarko** [comment]
Bizarre! Ok, I think I figured out the reason for it. It looks like toggling the bookmarks toolbar (ctrl-shift-B for me) causes this problem, at least for Chromium: _not_ having the bookmarks bar shown makes things look wrong, while having the bookmarks bar shown makes things look normal. It seems to be a persistent thing, since refreshing the page while in the "no bookmarks bar" state still doesn't fix the problem:

![whatthe](https://user-images.githubusercontent.com/4177727/95618451-ef4e0b80-0a21-11eb-8c84-842f31492c99.gif)

It turns out the problem is that, in some cases, a vertical scrollbar (for me this is a _second_ scrollbar, next to the one Emperor sometimes creates within its side panel) can appear on the right side of the screen -- and this scrollbar seems to be what pushes over Emperor into Empress' territory. Trying to replicate this condition is kind of tricky, but it looks like 1) the bookmarks bar going away 2) while Chromium is maximized 3) and when the view.qiime2.org header is shown does it consistently for me.

Ended up being solvable by adding `overflow: hidden;` to the `#emperor-in-empire` div, see below.

![fix](https://user-images.githubusercontent.com/4177727/95621185-22929980-0a26-11eb-9b74-7dae0453eb88.gif)

So, uh... not sure why that impacted me specifically but it should be fixed now :P PR incoming soon.

(For what it's worth, on my computer Firefox looked fine regardless of what I did with the bookmarks bar --)

![ff](https://user-images.githubusercontent.com/4177727/95618844-8e730300-0a22-11eb-80da-38eb72e3116c.gif)
